rest_framework_simplejwt/serializers.py

```diff
@@ -1,11 +1,16 @@
+from django.conf import settings
 from django.contrib.auth import authenticate
 from django.utils.translation import gettext_lazy as _
 from rest_framework import exceptions, serializers
+from rest_framework.exceptions import ValidationError
 
+from .exceptions import TokenError
 from .settings import api_settings
 from .state import User
 from .tokens import RefreshToken, SlidingToken, UntypedToken
 
+if 'rest_framework_simplejwt.token_blacklist' in settings.INSTALLED_APPS:
```

Comment:
Try this instead of checking INSTALLED_APPS: https://github.com/SimpleJWT/django-rest-framework-simplejwt/blob/master/rest_framework_simplejwt/serializers.py#L105-L112 It's a little better checking if blacklist is even turned on rather than checking if the app is installed.